README.md

```diff
@@ -1,2 +1,57 @@
+[![Build Status](https://travis-ci.org/dockstore/data-object-service-plugin.svg?branch=master)](https://travis-ci.org/dockstore/data-object-service-plugin)
+[![Coverage Status](https://coveralls.io/repos/github/dockstore/data-object-service-plugin/badge.svg?branch=master)](https://coveralls.io/github/dockstore/data-object-service-plugin?branch=master)
+
 # data-object-service-plugin
-Dockstore Data Object Service file provisioning plugin
+Dockstore Data Object Service file preprovisioning plugin
+
```

Comment:
Can you please include a link to the DOS repo, https://github.com/ga4gh/data-object-service-schemas